static/css/style.css

```diff
@@ -33,7 +33,22 @@ footer {
 	list-style-type: none;
 	margin-right: 200px;
 	text-align: auto;
+	font-size: 18px;
 }
+.h4-footer {
+	font-size: 20px;
+	border-bottom: 1px solid #CCCCCC;
+	border-left-width: 100px;
+	margin-bottom: 0.5em;
+	padding-bottom: 0.5em;
+}
+#map-canvas {
```

Comment:
I get it now, thx ;) This is a copt/paste from web, now I know i have to check also tabs :) Will correct it